content/post/ipfs-mobile-design-research

```diff
@@ -0,0 +1,45 @@
+---
+date: 2020-03-30
```

Comment:
just a reminder to update before posting =]

Reply:
Changed!